webapp/channels/src/components/admin_console/system_users/system_users_dropdown/system_users_dropdown.tsx

```diff
@@ -84,8 +84,12 @@ export default class SystemUsersDropdown extends React.PureComponent<Props, Stat
         };
     }
 
-    handleMakeActive = (e: {preventDefault: () => void}) => {
+    handleMakeActive = (e: React.MouseEvent<HTMLButtonElement>, disableActivationToggle: boolean) => {
         e.preventDefault();
+        e.stopPropagation();
+        if (disableActivationToggle) {
+            return;
+        }
```

Comment:
Is that extra check really necessary? If the user can't be disabled, the code should never reach this line. And if it does reach it, it might just fail instead of returning magically.

Reply:
having the extra check doesn't hurt to prevent potential dom manipulation.